packages/edit-site/src/components/editor/global-styles-renderer.js

```diff
@@ -39,11 +39,13 @@ function getBlockPresetsDeclarations( blockPresets = {} ) {
 		PRESET_METADATA,
 		( declarations, { path, valueKey, cssVarInfix } ) => {
 			const preset = get( blockPresets, path, [] );
-			preset.forEach( ( value ) => {
-				declarations.push(
-					`--wp--preset--${ cssVarInfix }--${ value.slug }: ${ value[ valueKey ] }`
-				);
-			} );
+			if ( preset ) {
```

Comment:
That's interesting. It looks like `get` doesn't add enough safety because `preset` is defined but set to some falsey value.